thruster_control/src/thruster_control.cpp

```diff
@@ -137,22 +137,6 @@ ThrusterControl::ThrusterControl(ros::NodeHandle& nodeHandle)
       publisher_reportBatteryHealth.add_check<diagnostic_tools::PeriodicMessageStatus>(
           "rate check", paramsReportsBatteryHealthCheckPeriod));
 
-  diagnostic_tools::MessageStagnationCheckParams paramsBatteryMessageStagnationcheck;
-  diagnostic_tools::Diagnostic diagnosticBatteryInfoStagnate(
-      diagnostic_tools::Diagnostic::WARN, ReportFault::BATTERY_INFO_STAGNATED);
-  paramsBatteryMessageStagnationcheck.stagnation_diagnostic(diagnosticBatteryInfoStagnate);
-
-  diagnosticsUpdater.add(
-      publisher_reportBatteryHealth.add_check<diagnostic_tools::MessageStagnationCheck>(
-          "stagnation check",
-          [batteryCurrentSteadyBand](const sensor_msgs::BatteryState& a,
-                                     const sensor_msgs::BatteryState& b)
-          {
-            return std::fabs(a.current - b.current) < batteryCurrentSteadyBand;
-          }
-          , paramsBatteryMessageStagnationcheck));
```

Comment:
@patapufate mind to explain why this is removed?

Reply:
The LEM granularity is too low so we get the same reading over and over while not spinning the thruster. This means it trips the error even in safe use cases. Same with the temperature sometimes.

Reply:
Oh, I see. Thanks for the explanation.

Reply:
We could keep the check with a zero steady band, but I don't mind strongly.